brainscore_language/artificial_subject.py

```diff
@@ -99,7 +99,12 @@ class RecordingTarget:
 
     class RecordingType:
         """ method of recording """
-        spikerate_exact = "spikerate_exact"  # the exact spike-rate activity of each neuron
+
+        spikerate_exact = "spikerate_exact"
+        """ the exact spike-rate activity of each neuron """
+
```

Comment:
how do we support future benchmarks that require a modality that is not explicitly specified? the recording types on the neural data side seem to be mutually exclusive for the most part (i.e. we don't have both spikerate and fMRI effect size in the same data). in that case, would be useful to have a default abstract entity, i.e. `neuroid` that we use by default in the absence of an explicit specification.

Reply:
if a new benchmark requires a new modality, they would have to change the `ArtificialSubject` declaration, i.e. send in a PR to change.